### ISSUE TITLE
Hyperlink styling/hover effect

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/techdocs#230]
Hyperlinks currently do not have a hover effect. Give hyperlinks an underline when the user hovers over them. The color of the underline should be the same as the hyperlink.

![image](https://user-images.githubusercontent.com/40569793/67710157-cd38b600-f995-11e9-8fb2-24620625d3e5.png)

Related bug:
Hyperlinks that are bold are black instead of green, making it impossible to tell they are hyperlinks. Make bold hyperlinks the same green as regular hyperlinks 
![image](https://user-images.githubusercontent.com/40569793/67710246-f8bba080-f995-11e9-8e69-84b2edc589bb.png)

**Comment by m4tr1k** [comment]
@Michael-Grover can I take a look at this one?

**Comment by Michael-Grover** [comment]
@m4tr1k yes, thank you!

**Comment by Michael-Grover** [comment]
The black hyperlink bug may have something to do with if it's in a bulleted list, I'm not sure

**Comment by Michael-Grover** [comment]
examples of black bold hyperlinks on this page:
https://docs.accordproject.org/docs/next/logic-ergo.html
